CustomApps/lyrics-plus/Utils.js

```diff
@@ -146,7 +146,7 @@ const Utils = {
 	rubyTextToReact(s) {
 		const react = Spicetify.React;
 
-		const rubyElems = s.split("<ruby>");
+		const rubyElems = s?.split("<ruby>");
```

Comment:
There is no point in having optional chaining here: if this ever resolves to undefined, rubyElems[0] will throw an error. It's better to just make sure that whatever is being passed to rubyTextToReact from processTranslatedLyrics is not undefined.